src/libraries/Microsoft.PowerFx.Core/Public/Values/CollectionTableValue.cs

```diff
@@ -95,6 +95,25 @@ public override async Task<DValue<RecordValue>> AppendAsync(RecordValue record,
             return DValue<RecordValue>.Of(record);
         }
 
+        public override async Task<DValue<BooleanValue>> ClearAsync(CancellationToken cancellationToken)
+        {
+            cancellationToken.ThrowIfCancellationRequested();
+
+            if (_sourceList == null)
+            {
+                return await base.ClearAsync(cancellationToken);
+            }
+
+            _sourceList.Clear();
+
+            if (_sourceList.Count == 0)
+            {
+                return DValue<BooleanValue>.Of(New(true));
+            }
+
+            return DValue<BooleanValue>.Of(New(false));
```

Comment:
We should ignore return value. 

[https://learn.microsoft.com/en-us/power-platform/power-fx/reference/function-clear-collect-clearcollect#clear](https://learn.microsoft.com/en-us/power-platform/power-fx/reference/function-clear-collect-clearcollect#clear)